src/test/java/fi/metatavu/metamind/test/functional/StoryTestsIT.java

```diff
@@ -59,7 +59,7 @@ public void testUpdateStory() throws Exception {
       builder.admin().stories().assertStoriesEqual(createdStory, builder.admin().stories().findStory(createdStory.getId()));
 
       Story updateStory = new Story();
```

Comment:
No need to create new story when it's overridden in next line, use:

    Story updateStory = builder.admin().stories().findStory(createdStory.getId());
    updateStory.setName("new name");